packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -170,61 +184,32 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
       PaidStreamingExtensionEvents.REQUEST,
       async (index: number, size: number, response: (allow: boolean) => void) => {
         const reqPrice = bigNumberify(size).mul(WEI_PER_BYTE);
-        const {peerAccount, peerOutcomeAddress} = wire.paidStreamingExtension;
-        const knownPeerAccount = this.peersList[torrent.infoHash][peerAccount];
-
-        if (!knownPeerAccount) {
-          log.info(
-            `>> wire first_request of ${peerAccount} with outcomeAddress ${peerOutcomeAddress} and leecherBalance ${WEI_PER_BYTE.mul(
-              torrent.length
-            )}`
-          );
-          const {channelId} = await this.paymentChannelClient.createChannel(
-            this.pseAccount, // seeder
-            peerAccount, // leecher
-            hexZeroPad(INITIAL_SEEDER_BALANCE.toHexString(), 32), // seederBalance,
-            hexZeroPad(WEI_PER_BYTE.mul(torrent.length).toHexString(), 32), // leecherBalance,
-            this.paymentChannelClient.myEthereumSelectedAddress, // seederOutcomeAddress,
-            peerOutcomeAddress // leecherOutcomeAddress
-          );
-          // eslint-disable-next-line require-atomic-updates
-          wire.paidStreamingExtension.pseChannelId = channelId;
+        const {pseChannelId, peerAccount: peer} = wire.paidStreamingExtension;
+        const knownPeer = this.peersList[torrent.infoHash][peer];
```

Comment:
It's not possible, because by the time this event can happen, this one https://github.com/statechannels/monorepo/blob/d30fb32bfee3463198aa88d2252c5f47ebf5e261/packages/web3torrent/src/library/web3torrent-lib.ts#L329 must happen first